source/form.html

```diff
@@ -25,12 +25,12 @@
           </ul>
           <!-- /.site-list -->
 
-          <div class="nav__control">
-            <button class="nav__button nav__button--search " type="button" aria-label="Поиск по сайту"></button>
+          <div class="nav__search search">
+            <button class="search__button search__button--search " type="button" aria-label="Поиск по сайту"></button>
```

Comment:
модификатор здесь тебе вообще не нужен
```suggestion
            <button class="search__button" type="button" aria-label="Поиск по сайту"></button>
```